File: object-oriented-programming/virtual-world-java/src/Game/Model/Population.java
package Game.Model;

import Game.Model.Animals.*;
import Game.Model.Plants.*;

public class Population {
    public static void populate(World world){
        new Wolf(world);
        new SowThistle(world);
        new Fox(world);
        new Guarana(world);
        new Turtle(world);
        new Belladonna(world);
        new Antelope(world);
        new SosnowskysHogweed(world);
        for(int i = 0; i < 2; i++){
            new Sheep(world);
            new Grass(world);
        }
    }
}

File: object-oriented-programming/virtual-world-python/population.py
from animals.wolf import Wolf
from plants.grass import Grass
from animals.sheep import Sheep
from plants.sow_thistle import Sow_Thistle
from animals.fox import Fox
from plants.guarana import Guarana
from animals.turtle import Turtle
from plants.belladonna import Belladonna
from animals.antelope import Antelope
from plants.sosnowskys_hogweed import Sosnowskys_Hogweed
from animals.cybersheep import Cybersheep

class Population :
    def __init__(self, world):
        for i in range(2) :
            Sheep(world)
            Turtle(world)
            Antelope(world)
            Fox(world)

        Wolf(world)
        Grass(world)
        Sow_Thistle(world)
        Guarana(world)
        Belladonna(world)
        Sosnowskys_Hogweed(world)
        Cybersheep(world)